blinky-statik/src/main/java/org/spideruci/analysis/statik/AnalysisConfig.java

```diff
@@ -80,7 +97,7 @@ private void parse() {
   }
   
   private boolean isLineComment(final String line) {
-    return line.startsWith("#") && line.startsWith("//");
+    return line.startsWith("#") || line.startsWith("//");
```

Comment:
Thank you 💯  ... this was clearly an obvious bug on my part. And this speaks to the value of code reviews.